search-commons/src/main/java/no/unit/nva/search/importcandidate/ImportCandidateParameter.java

```diff
@@ -206,6 +206,11 @@ public String errorMessage() {
         return errorMsg;
     }
 
+    @Override
+    public ParameterKey subquery() {
+        return null;
+    }
+
```

Comment:
Why does this return `null`, which is then cast to parameterkey elsewhere?

Nitpick: Rename to subQuery?

Reply:
nitPick is acceptable here :)
It is not implemented in ImportCandidate,